packages/flame/lib/src/test_helpers/canvas_commands/image_command.dart

```diff
@@ -0,0 +1,20 @@
+import 'dart:ui';
+import 'command.dart';
+
+/// canvas.drawImage()
```

Comment:
I don't compare the actual image, see comment on the `mockCanvas.drawImage` impl